cmd/docsync/main.go

```diff
@@ -0,0 +1,169 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"mime"
+	"os"
+	"path"
+	"path/filepath"
+	"sort"
+	"strings"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/s3"
```

Comment:
Before we land this PR, we're gonna want a separate PR that vendors the AWS s3 package, since (I think?) it hasn't been vendored yet. I haven't gone through the vendoring workflow in its current version, but there are instructures [here](https://github.com/chain/chain#dependencies), and maybe we can work together on that.